diff --git a/rules/modules/path.rules.inc b/rules/modules/path.rules.inc
index e1328fc..5bff19a 100644
--- a/rules/modules/path.rules.inc
+++ b/rules/modules/path.rules.inc
@@ -30,7 +30,7 @@ function path_rules_condition_info() {
  * Condition implementation: Check if the path has an alias
  */
 function rules_condition_url_has_alias($settings) {
-  return (bool)drupal_lookup_path('alias', $settings['src'], $settings['language']);
+  return (bool)drupal_lookup_path('alias', $settings['src'], isset($settings['language']) ? $settings['language'] : '');
 }
 
 /**
@@ -38,7 +38,7 @@ function rules_condition_url_has_alias($settings) {
  */
 function rules_condition_alias_exists($settings) {
   $dst = rules_clean_path($settings['dst'], $settings['replace']);
-  return (bool)drupal_lookup_path('source', $dst, $settings['language']);
+  return (bool)drupal_lookup_path('source', $dst, isset($settings['language']) ? $settings['language'] : '');
 }
 
 /**
